buildSrc/src/main/rocker/com/uber/okbuck/template/android/ManifestRule.rocker.raw

```diff
@@ -9,8 +9,7 @@ String versionName,
 boolean debuggable,
 Collection secondaryManifests
 )
-okbuck_manifest(
-    name = '@name',
+@com.uber.okbuck.template.base.BuckRule.template() -> {
```

Comment:
same as okbuck_keystore here. 

If you want to proceed with these changes, you'll have to carefully check rulekey changes on the repo that uses OkBuck to verify there are no unexpected changes.